src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/inventory/nbt/NBTTagList.kt

```diff
@@ -27,7 +27,10 @@ class NBTTagList(override val rawNBT: MCNbtList) : NBTBase(rawNBT) {
         rawNBT.add(index, nbt)
     }
 
-    fun removeTag(index: Int) = rawNBT.removeAt(index)
+    // TODO(breaking): Return self instead of the removed element
+    fun removeTag(index: Int) = apply {
+        rawNBT.removeAt(index)
+    }
```

Comment:
Not sure if I like this, this is a pretty typical index-removal API. Is there a specific reason for this change?

Reply:
Should I wrap the returned element or leave it as is?

Reply:
Yeah good point, wrapping makes sense